content/lxd/introduction.md

```diff
@@ -1,87 +1,116 @@
+  * [Virtual machines](https://linuxcontainers.org/lxd/docs/master/virtual-machines) (implemented through the use of qemu)
+  * [Configurable through profiles](https://linuxcontainers.org/lxd/docs/master/profiles) (applicable to both containers and virtual machines)
+
+Backup and export
+: * [Backup and recovery](https://linuxcontainers.org/lxd/docs/master/backup) (for all objects managed by LXD)
+  * [Snapshots](https://linuxcontainers.org/lxd/docs/master/instances#snapshot-scheduling) (to save and restore the state of an instance)
+  * [Container and image transfer](https://linuxcontainers.org/lxd/docs/master/image-handling) (between different hosts, using images)
+  * [Live migration](https://linuxcontainers.org/lxd/docs/master/migration) (using CRIU)
+
+Configurability
+: * [Multiple storage backends](https://linuxcontainers.org/lxd/docs/master/storage) (with configurable storage pools and storage volumes)
+  * [Network management](https://linuxcontainers.org/lxd/docs/master/networks) (including bridge creation and configuration, cross-host tunnels, ...)
+  * [Advanced resource control](https://linuxcontainers.org/lxd/docs/master/instances#resource-limits-via-limitskernellimit-name) (CPU, memory, network I/O, block I/O, disk usage and kernel resources)
+  * [Device passthrough](https://linuxcontainers.org/lxd/docs/master/container-environment) (USB, GPU, unix character and block devices, NICs, disks and paths)
 
-To learn more about LXD and OpenNebula, take a look at our [getting started guide](/lxd/getting-started-opennebula/).
 
 # Availability
 LXD works on any recent Linux distribution. LXD upstream directly maintains the Ubuntu packages
 and also publishes a snap package which can be used with most of the popular Linux distributions.
 
-More details can be found on our [getting started](/lxd/getting-started-cli/) page.
-
-# Relationship with LXC
-LXD isn't a rewrite of LXC, in fact it's building on top of LXC to provide a new,
-better user experience. Under the hood, LXD uses LXC through liblxc and its Go binding
-to create and manage the containers.
+In addition, LXD is available for Windows and macOS.
 
-It's basically an alternative to LXC's tools and distribution template system
-with the added features that come from being controllable over the network.
-
-# Language, licensing and contributions
-LXD is written in Go, it's free software and is developed under the Apache 2 license.
+More details can be found on our [getting started](/lxd/getting-started-cli/) page.
 
-There are no CLA or similar legal agreements required to contribute to LXD,
-however we do require commits be signed-off (following the DCO - Developer Certificate of Ownership).
+[<img src="/static/img/snapstore.svg" alt="Snapstore logo" class="logo-line"/>](https://snapcraft.io/store)
+[<img src="/static/img/chocolatey.svg" alt="Chocolatey logo" class="logo-line"/>](https://chocolatey.org/)
```

Comment:
@ru-fu can you put sizes on these to restrict them in github view?

Reply:
Not a fan of that since it's bad web programming, but it's the only way to make them show up ok in GitHub - so I guess we don't have a choice. :(